apps/codesandbox-react-template/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "codesandbox-react-template",
+  "description": "Template for creating codesandbox environment for @fluenui/react* packages.",
+  "version": "7.0.0",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "lint": "just-scripts lint",
+    "test": "just-scripts test",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "start": "just-scripts dev"
+  },
+  "devDependencies": {
+    "@types/es6-promise": "0.0.32",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/tslint-rules": "^7.2.1",
+    "@uifabric/build": "^7.0.0"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.13",
+    "@microsoft/load-themed-styles": "^1.10.26",
+    "es6-promise": "^4.1.0",
+    "immutability-helper": "~2.8.1",
+    "office-ui-fabric-react": "^7.121.1",
```

Comment:
minor nit; I would suggest this depends on `@fluentui/react` instead of `office-ui-fabric-react`.